Create Innovation

hhartingrex opened the pull request:
First draft on a new page under Project Management

<!-- readthedocs-preview civicactions-handbook start -->
----
:books: Documentation preview :books:: https://civicactions-handbook--1076.org.readthedocs.build/en/1076/

<!-- readthedocs-preview civicactions-handbook end -->

hhartingrex committed:
Create Innovation

First draft on a new page under Project Management

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

hhartingrex committed:
Update Innovation

Small typo fixes. 
NOTE: I am doing something goofy, because I cannot see this file in Preview. I will need to learn why (as I suspect it requires a fix!)

hhartingrex committed:
Update Innovation

Fixed link (hopefully!)

dmundra committed:
Merge branch 'master' into hhartingrex-patch-4

farooqzakhilwal committed:
Update Innovation

farooqzakhilwal reviewed (approved):
looks great. Just minor edits. @hhartingrex leaving this for you to merge.

dmundra commented on practice-areas/project-management/Innovation:
@hhartingrex looks like the filename needs to be 'innovation.md' so that the page loads. Also since this is a new page it needs to be added to `https://github.com/CivicActions/guidebook/blob/master/.config/mkdocs.yml#L170`

dmundra reviewed (changes requested):
Need changes to the filename and added to the navigation code block.

hhartingrex committed:
Rename Innovation to innovation

hhartingrex committed:
Hhartingrex patch 2 1 (#1079)

* Update project-management-checklists.md (#1068)

* Update project-management-checklists.md

Small addition to the text as per the suggestion of the team.

* [pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

* Update project-management-checklists.md

Denise adjustments included. Thank you!

---------

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
Co-authored-by: Farooq Zakhilwal <73595515+farooqzakhilwal@users.noreply.github.com>

* [pre-commit.ci] pre-commit autoupdate (#1075)

updates:
- [github.com/codespell-project/codespell: v2.2.2 → v2.2.4](https://github.com/codespell-project/codespell/compare/v2.2.2...v2.2.4)

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

* Update mkdocs.yml

Adjusted Project Management menu to include Innovation. Changed Help Desk to Support.

---------

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
Co-authored-by: Farooq Zakhilwal <73595515+farooqzakhilwal@users.noreply.github.com>

hhartingrex committed:
Merge branch 'master' into hhartingrex-patch-4

hhartingrex committed:
Rename innovation to innovation.md

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci
